### ISSUE TITLE
VPack as block storage for SQlite

### ISSUE THREAD

**Issue by dumblob** [arangodb/velocypack#99]
Just curious - has anyone tried on some project using VelocyPack as the backend storage for SQlite? Any benchmarks or experience or lessons learned?

**Comment by dothebart** [comment]
Interesting Idea. 
No, we haven't heard of somebody doing this. 
One implementation problem could be, that you need a C interface if I grasp this correctly?
Sqlite is pure C if I get this right? VPack otoh is implemented in C++.

**Comment by dumblob** [comment]
> One implementation problem could be, that you need a C interface if I grasp this correctly?
> Sqlite is pure C if I get this right? VPack otoh is implemented in C++.

Yes, SQlite is pure C and VPack is C++. But I somehow assumed the C++ implementation offers also a C interface. If it's not the case, then I'm pretty sure nobody used VPack as SQlite backend because that'd be much more work than "just plumbing".
